src/main/modules/helmet/index.ts

```diff
@@ -6,6 +6,8 @@ export interface HelmetConfig {
 }
 
 const self = "'self'";
+const view_report_script_1 = "'sha256-jVI3dCx2J6gL6bzR1ljQZftjkTtjif8MLnKuMb2CK4w='";
```

Comment:
What's that? :)

Reply:
The scripts on the page cannot run unless I allow them to because of the Content Security Policy. It doesn't happen in dev mode so thats what Arun had issues.
The sha is the scripts hash, and apparently you have to allow them explicitly.
Incidentally there are some scripts from the Gov Uk Nunjucks that are being blocked because of the same issue.

Reply:
Browser Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-8x0U8ucWIxYD4ShTA5KJD1fGCerO0FA90OiUpT5Y5b0='), or a nonce ('nonce-...') is required to enable inline execution.

Reply:
yeah that is the second SHA, there are two I added

Reply:
Is that because those scripts are inline?

Reply:
Possibly, but moving it to a separate file might also have its issues

Reply:
yeah so the inline is being blocked as unsafe-inline and a separate file would result in unsafe-eval. I would have to allow both due to the CSP

Reply:
Are we in control of the CSP?

Reply:
yeah, thats what I am adding the hashes to in that file.